comdb2/_cdb2_types.py

```diff
@@ -9,21 +9,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from collections import namedtuple
-from datetime import datetime
+from __future__ import annotations
 
```

Comment:
It's not necessary, but it can't just be removed, either. I'd need to rewrite annotations like:
```py
class DatetimeUs(datetime.datetime):
    @classmethod
    def fromdatetime(cls, dt: datetime.datetime) -> DatetimeUs:
```
to instead be:
```py
class DatetimeUs(datetime.datetime):
    @classmethod
    def fromdatetime(cls, dt: datetime.datetime) -> "DatetimeUs":
```
I'd rather not, though, since I find that harder to maintain. Do you strongly prefer avoiding `from __future__ import annotations`?

Reply:
That's fair, let's not remove it then.